### PR TITLE
Ep #6 Show Notes

### PR DESCRIPTION
**Pull request by bstrie**

### CODE REVIEW

File: _episodes/005-rust-1.38.0.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Rust in Production: An Interview with Armin Ronacher"
```
Comment (jonhoo) [inline _episodes/005-rust-1.38.0.md]: This doesn't seem right?

File: _episodes/005-rust-1.38.0.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+date: 2019-09-17T18:00:00+0000
```
Comment (jonhoo) [inline _episodes/005-rust-1.38.0.md]: This also doesn't seem right?

File: _episodes/005-rust-1.38.0.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+* [@1:15] - Pipelined compilation
+
```
Comment (jonhoo) [inline _episodes/005-rust-1.38.0.md]: I don't know if these spaces between the lines should be there? I think markdown requires consecutive lines to make them a list?

Comment (bstrie) [reply _episodes/005-rust-1.38.0.md]: Hm, they look fine in the rendered output on Github?

Comment (jonhoo) [reply _episodes/005-rust-1.38.0.md]: It's not actually rendered as _one_ list, it's rendered as multiple single-item lists. This probably doesn't matter a tonne, but it's not _quite_ right.

**Pull request merged**